frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -879,14 +879,20 @@ function FeatureFlagRollout({ readOnly }: FeatureFlagReadOnlyProps): JSX.Element
         <>
             {readOnly ? (
                 <>
-                    <div className="font-medium border rounded p-2 pl-3 mb-4">
-                        Served value:{' '}
+                    <div className="flex flex-col mb-4">
+                        <span className="card-secondary">Type</span>
                         <b>
                             {featureFlag.filters.multivariate
                                 ? 'Multiple variants with rollout percentages (A/B test)'
                                 : 'Release toggle (boolean)'}
                         </b>
+                        <span className="card-secondary mt-4">Flag persistence</span>
+                        <b>
+                            This flag {featureFlag.ensure_experience_continuity ? 'persists' : 'does not persist'}{' '}
```

Comment:
@clarkus at a glance, it seems hard to distinguish whether persistence is enabled or not just by reading the sentence.

Does it make sense to not show this bit at all when persistence is disabled?

Reply:
I was erring towards describing the state regardless of the setting. An absence of that info could be seen as less clear - you'd have to enter the edit view to find out for sure. It might be something to test with users, but we could also consider something more visual to make the setting easier to scan.

Reply:
I updated https://www.figma.com/file/eX6AKgqX2bZDxgEg2S4xzy/Feature-Flags-%26-Experiments?node-id=4%3A9727. This removes the bold / emphasis from the values of each attribute and instead only emphasizes the "does not persist" portion of the value.

<img width="472" alt="Screen Shot 2022-09-15 at 10 47 15 AM" src="https://user-images.githubusercontent.com/254612/190474506-e5d7e0b8-5c29-47f8-a059-fad6faf40784.png">